R/utility.R

```diff
@@ -267,36 +289,36 @@ createOutputFile <- function(filePattern, url){
       uploadCondition = "taskCompletion"
     )
   )
-  
+
```

Comment:
Ok. In general I'm not super stoked about massive style changes since they cause a lot of churn and can cause lots of no-op merge conflicts on other branches. Lets sync up on the linter requirements and be sure to add it to the documentation for contributors.